en/guides/publishing-open-source-code.md

```diff
@@ -30,7 +30,7 @@ The updates to the Directive on Management of IT support the Digital Standard #3
 
 Aligned with the Open Government vision, teams should by default consider adapting their process to develop as open source from the inception of projects to reduce the overhead required to release their source code later down the road.
 
-It has also been found that doing so improves the quality of the code developed and encourages collaboration. See this [blog post](https://mojdigital.blog.gov.uk/2017/02/21/why-we-code-in-the-open/) from the United Kingdom Ministry of Justice for more details.
+It has also been found that doing so improves the quality of the code developed and encourages collaboration.
```

Comment:
If we're saying that, then we need to point to where it was found? Otherwise I can say whatever I want: OSS has proven to support people to fly without wings. :wink:

Reply:
Right I was removing it because it's an external link that's not bilingual

Reply:
Makes sense!